install/operator/pkg/generator/generator.go

```diff
@@ -30,6 +30,21 @@ func AssetAsBytes(path string) ([]byte, error) {
 	return prometheusRules, nil
 }
 
+func IsDirectory(path string) bool {
```

Comment:
Why making this public when it is used only in this package?

Reply:
It was used outside the package in a previous incarnation so left as pulbic as 'might be useful in the future'. Can make private if preferred.